oss-vienna-happy-hour.mdx

```diff
@@ -0,0 +1,232 @@
+
+At Xata, we are passionate about PostgreSQL and its role in powering modern applications.
+We believe that PostgreSQL is more than just a database; it is a fundamental technology that enables innovation and growth in the tech industry.
+And we are excited to celebrate this awesome project with the Postgres community at the Open Source Summit Europe!
+
+The event is free and open to everyone, even if you are not attending the Open Source Summit.
+However, space is limited to 50 attendees, so please fill out[this form](https://5i8caik7lja.typeform.com/psql-happy-hour) if you'd like to attend.
+
+If you're a Postgres fan and in the area, you're very welcome to join us!
+
+### Why attend?
+
+- **Engaging conversations**: Whether you’re a Postgres expert or curious about how Xata can elevate your next project, connect with like-minded individuals and explore new possibilities.
+- **Exclusive swag**: Attendees will receive some cool Xata swag as a token of appreciation.
+- **Relaxed atmosphere**: We believe the best ideas flow in a casual setting. Whether you’re diving deep into database architecture or just starting out, expect lively discussions, knowledge sharing, and maybe even a friendly debate or two.
+
+Because talking databases is always better with a drink in hand—come network, learn, and enjoy the Postgres Happy Hour!
+
+In the spirit of celebrating Postgres, we will be adhering to the community's [code of conduct](https://www.postgresql.org/about/policies/coc/).
+
+### A personal Postgres story
+
+The passage is coherent and reads well, conveying a personal and relatable story. However, a few minor edits can improve the flow and clarity:
+
+1. **Punctuation and Sentence Structure**: Adding commas where needed and breaking down a longer sentence for better readability.
+2. **Clarity**: Clarifying the timeline in the first sentence for smoother comprehension.
+
+Here's a revised version:
+
+---
+
+### A Personal Postgres Story
+
+When we were planning the Postgres Happy Hour in Vienna, I realized that it was almost 9 years ago, around the same time of year, that I gave my first PGConf.EU talk in Vienna.
+I was overwhelmed by the size of the room and how crowded it felt, despite its spaciousness. It was my first Postgres talk in English, and delivering it at the biggest Postgres conference was terrifying.
+The talk was about [Managing Postgres with Ansible](https://slides.com/apatheticmagpie/managing-postgres-with-ansible), which was a cool topic at the time. I remember making so many slides and forgetting to breathe. No other talk seemed so hard after that—it was a big learning experience.
+
+![Gulcin and Slonik, in Vienna in 2015](/images/oss-vienna/gulcin-with-slonik-2015.jpg)
```

Comment:
This needs to be resized outside (usually just preview on mac) and then re-uploaded so its not so huge :)

Reply:
Done, let's see how this looks :)